addons/website/models/res_config.py

```diff
@@ -16,37 +16,84 @@ def _default_website(self):
+    social_youtube = fields.Char("Youtube", related='website_id.social_youtube')
+    social_googleplus = fields.Char("Google+", related='website_id.social_googleplus')
+    cdn_activated = fields.Boolean('Content Delivery Network (CDN)', related='website_id.cdn_activated')
     cdn_url = fields.Char(related='website_id.cdn_url')
     cdn_filters = fields.Text(related='website_id.cdn_filters')
-    module_website_form_editor = fields.Selection([
-        (0, 'Use standard forms'),
-        (1, 'Create and customize forms to generate emails, leads, issues and extra information in the checkout process (new snippet available)')
-    ], "Form Builder")
-    module_website_version = fields.Selection([
-        (0, 'No version management and A/B testing (easy)'),
-        (1, 'Allow multiple versions of the same page (advanced)')
-    ], "A/B Testing")
+
+    module_website_form_editor = fields.Boolean("Custom Forms")
+    module_website_version = fields.Boolean("A/B Testing")
+    module_website_twitter = fields.Boolean("Twitter Roller")
+    module_website_blog = fields.Boolean("Blogs")
+    module_website_livechat = fields.Boolean("Live Chat")
+    module_website_forum = fields.Boolean("Forum")
+    module_website_crm = fields.Boolean("Contact Form")
+    module_website_slides = fields.Boolean("Slides")
+    module_website_hr_recruitment = fields.Boolean("Jobs")
+    module_website_sale = fields.Boolean("eCommerce")
+    module_website_contract = fields.Boolean("Subscriptions")
+    module_website_event_sale = fields.Boolean("Event Tickets")
+
     favicon = fields.Binary('Favicon', related='website_id.favicon')
 
     # Set as global config parameter since methods using it are not website-aware. To be changed
     # when multi-website is implemented
     google_maps_api_key = fields.Char(string='Google Maps API Key')
 
+    has_google_analytics = fields.Boolean(
+        "Google Analytics",
+        compute='_compute_has_google_analytics',
+        readonly=False,
+        required=True)
+
+    has_google_analytics_dashboard = fields.Boolean(
+        "Google Analytics in Dashboard",
+        compute='_compute_has_google_analytics_dashboard',
+        readonly=False,
+        required=True)
+
+    has_google_maps = fields.Boolean(
+        "Google Maps",
+        compute='_compute_has_google_maps',
+        readonly=False,
+        required=True)
+
     def set_google_maps_api_key(self):
         self.env['ir.config_parameter'].set_param(
             'google_maps_api_key', (self.google_maps_api_key or '').strip(), groups=['base.group_system'])
 
     def get_default_google_maps_api_key(self, fields):
         google_maps_api_key = self.env['ir.config_parameter'].get_param('google_maps_api_key', default='')
         return dict(google_maps_api_key=google_maps_api_key)
+
+    @api.depends('language_ids')
+    def _compute_number_of_languages(self):
+        for wizard in self:
```

Comment:
I would use config instead of wizard. Wizard is an old name, when we had wizards only in memory. Now those are classic models, so let us use a real variable name like config, or website_config if you like long names.